z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -281,26 +281,26 @@ cd ocp_upi
+| `pullsecret` | \<pull-secret\> | Get from [cloud.redhat.com](https://console.redhat.com/openshift/install/ibmz/user-provisioned)                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| `sshkey` | \<ssh-key\>| The SSH public key for the core user in RHEL CoreOS                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| `os_dns_domain` | \<external DNS ip addr\> or \<bastion ip addr\>| If you want to use your external or existing DNS server set `os_dns_domain` to use it, others set bastion machine ip address                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| `cluster_name` | \<cluster-name\> | The name of the cluster, such as `openshift`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | 
+| `base_domain` | \<cluster-base-domain\> | The base domain of the cluster, the base domain is used to create routes to your OpenShift Container Platform cluster components, such as `example.com`                                                                                                                                                                                                                                                                                                                                                                                               | 
```

Comment:
Please add new `update_bastion` parameter into README doc.

Reply:
Thanks @shhull. I add this part on L333 in `optional` section.